lld/COFF/Driver.cpp

```diff
@@ -2338,6 +2338,9 @@ void LinkerDriver::linkerMain(ArrayRef<const char *> argsArr) {
       config->is64() &&
       args.hasFlag(OPT_highentropyva, OPT_highentropyva_no, true);
 
+  // Handle /nodbgdirmerge
+  config->mergeDebugDirectory = args.hasArg(OPT_nodbgdirmerge);
```

Comment:
Isn't this inverted? If the option is present, we should have _no_ debug dir merge, and we should set `megeDebugDirectory` to false?